File: backwardPrimesCW.ts
function isPrime(num: number){
    if (num < 2){
      return false;
    }
    let i: number = 2;
    while(i < num){
      if (num % i == 0){
        return false;
      }
      i++;
    }
    return true;
  }
  
  
  export function backwardsPrime(start: number, stop: number): number[] {
    let ans: number[] = [];
    for(start; start <= stop; start++){
      if(isPrime(start) && isPrime(parseInt(start.toString().split('').reverse().join(''))) && start != parseInt(start.toString().split('').reverse().join(''))){
        ans.push(start);
      }
    }
    
      return ans;
  }
  
  

File: backwardPrimesCW.rb
def isPrime(num)
    return false if num < 2
    (2...num).each do |i|
      return false if num % i == 0
    end
    return true
 end
  
def backwardsPrime(start, stop)
    ans = []
    (start..stop).each do |i|
      ans.push(i) if isPrime(i) && isPrime(i.to_s.reverse.to_i) && i != i.to_s.reverse.to_i
      
    end
    return ans
end